src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -18,6 +29,49 @@
     public static final String USERGUIDE_URL = "https://ay2021s2-cs2103t-t11-1.github.io/tp/UserGuide.html";
     public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
 
+    private static final int TABLE_ITEM_HEIGHT = 32;
+    private static final int ADDITIONAL_MARGIN = 30;
+    private static final double WINDOW_HEIGHT = 800;
+    private static final double WINDOW_WIDTH = 900;
+
+    /**
+     * Creates wrapping ability for cells in TableView.
+     */
+    private static final Callback<TableColumn<CommandHelper, String>, TableCell<CommandHelper, String>>
+            WRAPPING_CELL_FACTORY = new Callback<TableColumn<CommandHelper, String>,
+            TableCell<CommandHelper, String>>() {
+                @Override
+                public TableCell<CommandHelper, String> call(TableColumn<CommandHelper, String> param) {
+                    TableCell<CommandHelper, String> tableCell = new TableCell<CommandHelper, String>() {
+                        @Override
+                        protected void updateItem(String item, boolean empty) {
+                            if (item == getItem()) {
+                                return;
+                            }
+
+                            super.updateItem(item, empty);
+
+                            if (item == null) {
+                                super.setText(null);
+                                super.setGraphic(null);
+                            } else {
+                                super.setText(null);
+                                Label l = new Label(item);
+                                l.setWrapText(true);
+                                VBox box = new VBox(l);
+                                l.heightProperty().addListener((observable, oldValue, newValue) -> {
+                                    box.setPrefHeight(newValue.doubleValue() + 7);
+                                    Platform.runLater(() -> this.getTableRow().requestLayout());
+                                });
+                                l.setTextFill(Color.web("white"));
+                                super.setGraphic(box);
+                            }
+                        }
+                    };
+                    return tableCell;
+                }
+            };
+
```

Comment:
this large chunk of code here is to allow wrap text because TableView does not support wrapping at the parent level and I need to go down to the children